MdeModulePkg/Core/Dxe/Event/Event.c

```diff
@@ -649,19 +650,38 @@ CoreWaitForEvent (
   IN EFI_EVENT  *UserEvents,
   OUT UINTN     *UserIndex
   )
+{
+  if (gEfiCurrentTpl != TPL_APPLICATION) {
+    return EFI_UNSUPPORTED;
+  }
+
+  return CoreWaitForEventInternal (NumberOfEvents, UserEvents, UserIndex); // MU_CHANGE - Support all TPLs

```

Comment:
Isn't this already inside a mu_change block?

Reply:
Yes good call, I've removed!